static/ts/tsconfig.json

```diff
@@ -1,16 +1,23 @@
 {
-  "compilerOptions": {
-    "sourceMap": true,
-    "emitDecoratorMetadata": true,
-    "experimentalDecorators": true,
-    "removeComments": false,
-    "noImplicitAny": false,
-    "noFallthroughCasesInSwitch": true,
-    "suppressImplicitAnyIndexErrors": true,
-    "newLine": "lf",
-    "module": "commonjs",
-    "target": "es5",
-    "jsx": "react",
-    "allowJs": false
-  }
+    "compilerOptions": {
+        /* Basic options */
+        "target": "es5",
+        "module": "commonjs",
```

Comment:
I think we want to target `es6` modules in order for Webpack tree shaking to be effective.

Reply:
I don't know what this option does precisely and thus whether the following is relevant, but note that phantomjs (the browser used for our casper tests) doesn't support same basic ES6 like arrow functions.

Reply:
To be clear, I’m proposing only `"module": "es6"`, not `"target": "es6"`. Webpack understands ES6 modules and compiles them away for bundling.

Reply:
Ahh, OK, great.